framework/tlvf/test/main.cpp

```diff
@@ -34,40 +32,23 @@ using namespace mapf;
 
 int main(int argc, char *argv[])
 {
-    size_t hlen = cCmduHeader::get_initial_size() + tlvVendorSpecific::get_initial_size() +
-                  beerocks_message::cACTION_HEADER::get_initial_size();
-    std::cout << "NON1905: " << tlvNon1905neighborDeviceList::get_initial_size() << std::endl;
-    std::cout << "HEADER: " << hlen << std::endl;
-
+    int errors = 0;
     mapf::Logger::Instance().LoggerInit("TLVF example");
-
-    MAPF_INFO("initializing local bus interface");
-    LocalBusInterface bus(Context::Instance());
-    bus.Init();
+    uint8_t tx_buffer[256];
 
     //START BUILDING THE MESSAGE HERE
-    CmduTxMessage cmdu_tx_msg;
-
-    //setting metadata, not related to tlvf
-    memcpy(cmdu_tx_msg.metadata()->dst, "\x01\x80\xc2\x00\x00\x13", 6);
-    cmdu_tx_msg.metadata()->ether_type = ETH_P_1905_1;
-    cmdu_tx_msg.metadata()->msg_type   = 0x8019;
-    cmdu_tx_msg.metadata()->length =
-        256; //buffer size, needs to be bigger for longer messages, otherwise segfault or memory leak might occur
-    memset(cmdu_tx_msg.data(), 0x00, cmdu_tx_msg.metadata()->length);
+    memset(tx_buffer, 0, sizeof(tx_buffer));
```

Comment:
Actually might be better to remove this line.
TLVF should set the buffer to 0 by itself (on create), so skipping the memset can provide another level of sanity.

Reply:
I don't see why this should block the merge though - it doesn't hurt to clear the buffer..